bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/i18n/mynice.properties

```diff
@@ -31,22 +31,70 @@ thing-type.config.mynice.swing.id.description = ID of the gate on the TP4 bus co
+channel-type.mynice.doorstatus.state.option.CLOSING = Closing
+channel-type.mynice.doorstatus.state.option.STOPPED = Stopped
+channel-type.mynice.doorstatus.command.option.STOP = Stop
+channel-type.mynice.doorstatus.command.option.MOVE = Move
 channel-type.mynice.moving.label = Moving
 channel-type.mynice.moving.description = Indicates if the device is currently operating a command
 channel-type.mynice.obstruct.label = Obstruction
 channel-type.mynice.obstruct.description = Something prevented normal operation of the gate by crossing the infra-red barrier
 channel-type.mynice.t4command.label = T4 Command
 channel-type.mynice.t4command.description = Send a T4 Command to the gate
+channel-type.mynice.t4command.command.option.MDAx = Step by Step
+channel-type.mynice.t4command.command.option.MDAy = Stop (as remote control)
+channel-type.mynice.t4command.command.option.MDAz = Open (as remote control)
+channel-type.mynice.t4command.command.option.MDA0 = Close (as remote control)
+channel-type.mynice.t4command.command.option.MDA1 = Partial opening 1
+channel-type.mynice.t4command.command.option.MDA2 = Partial opening 2
+channel-type.mynice.t4command.command.option.MDA3 = Partial opening 3
+channel-type.mynice.t4command.command.option.MDBi = Apartment Step by Step
+channel-type.mynice.t4command.command.option.MDBj = Step by Step high priority
+channel-type.mynice.t4command.command.option.MDBk = Open and block
+channel-type.mynice.t4command.command.option.MDBl = Close and block
+channel-type.mynice.t4command.command.option.MDBm = Block
+channel-type.mynice.t4command.command.option.MDEw = Release
+channel-type.mynice.t4command.command.option.MDEx = Courtesy light timer on
+channel-type.mynice.t4command.command.option.MDEy = Courtesy light on-off
+channel-type.mynice.t4command.command.option.MDEz = Step by Step master door
+channel-type.mynice.t4command.command.option.MDE0 = Open master door
+channel-type.mynice.t4command.command.option.MDE1 = Close master door
+channel-type.mynice.t4command.command.option.MDE2 = Step by Step slave door
+channel-type.mynice.t4command.command.option.MDE3 = Open slave door
+channel-type.mynice.t4command.command.option.MDE4 = Close slave door
+channel-type.mynice.t4command.command.option.MDE5 = Release and Open
+channel-type.mynice.t4command.command.option.MDFh = Release and Close
+
+# channel types config
+
+channel-type.config.mynice.courtesy.duration.label = Duration
+channel-type.config.mynice.courtesy.duration.description = Duration the lamp stays on
+
+# channel types
+
+channel-type.mynice.doorstatus.state.option.open = Open
+channel-type.mynice.doorstatus.state.option.closed = Closed
+channel-type.mynice.doorstatus.state.option.opening = Opening
+channel-type.mynice.doorstatus.state.option.closing = Closing
+channel-type.mynice.doorstatus.state.option.stopped = Stopped
+channel-type.mynice.doorstatus.command.option.stop = Stop
+channel-type.mynice.doorstatus.command.option.move = Move
```

Comment:
This block has to be removed here.

Reply:
Clean and reruned properties generation.